### ISSUE TITLE
extract-text-webpack-plugin@2 support in extract-svg-plugin

### ISSUE THREAD

**Issue by farwayer** [kisenka/svg-sprite-loader#61]
```js
throw new Error("Breaking change: ExtractTextPlugin now only takes a single argument. Either an options " +
		^

Error: Breaking change: ExtractTextPlugin now only takes a single argument. Either an options object *or* the name of the result file.
Example: if your old code looked like this:
    new ExtractTextPlugin('css/[name].css', { disable: false, allChunks: true })

You would change it to:
    new ExtractTextPlugin({ filename: 'css/[name].css', disable: false, allChunks: true })

The available options are:
    filename: string
    allChunks: boolean
    disable: boolean
```

**Comment by kisenka** [comment]
Duplicate of #54

**Comment by kisenka** [comment]
Shipped with 0.0.30. Also now you can omit extract=true option, loader will try to detect extract mode automatically.
